### ISSUE TITLE
Build as UMD module

### ISSUE THREAD

**Issue by jasny** [ltonetwork/simple-iframe-rpc#1]
When building a project as [UMD module](https://github.com/umdjs/umd) we get the following error:

```
ERROR in simple-iframe-rpc.listener.js
simple-iframe-rpc.listener.js from Terser plugin
Unexpected token: keyword (var) [simple-iframe-rpc.listener.js:11,16]
    at js_error (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:545:11)
    at croak (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:1269:9)
    at token_error (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:1277:9)
    at unexpected (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:1283:9)
    at expr_atom (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:2560:9)
    at maybe_unary (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:3405:19)
    at expr_ops (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:3456:24)
    at maybe_conditional (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:3461:20)
    at maybe_assign (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:3538:20)
    at expression (/home/arnold/Projects/ltonetwork/simple-iframe-rpc/node_modules/terser/dist/bundle.min.js:3563:24)
```

_Note that all typescript projects by LTO Network are giving this error when trying to build as UMD._

**Comment by JoshuaVB** [comment]
@jasny The issue is from Terser minification.
Adding this to webpack.config fixes the error:
```
optimization: {
        minimize: false
      },
```

So then the next step is fixing the minimizer. We could also explore a different minification plugin:

```
optimization: {
        minimize: true,
        // could also explore using a different minification plugin depending on output
        minimizer: [new TerserPlugin({
            // add options to make work, probably
        })],
},
```

**Comment by jasny** [comment]
The webpack default shouldn't be failing. Otherwise everyone building as UMD would have this issue.

Is the an option in our config that's making minimalization fail?

**Comment by JoshuaVB** [comment]
@jasny That's fair, I've done some more digging and I think I've found the option that is causing your minimization to fail.

```
experiments: {
  outputModule: true,
},
```

When this is set to `false` the package compiles as UMD without issue. 
This is not default webpack, so I believe that's the culprit, I checked and the umd-test repo has the same thing.

Would removing outputModule from the webpack config be an acceptable solution to compile in UMD?
